app/src/main/java/org/mozilla/fenix/settings/quicksettings/WebsiteInfoView.kt

```diff
@@ -38,7 +38,11 @@ class WebsiteInfoView(
         bindUrl(state.websiteUrl)
         bindTitle(state.websiteTitle)
         bindSecurityInfo(state.securityInfoRes, state.iconRes, state.iconTintRes)
-        bindCertificateName(state.certificateName)
+        if (state.certificateName != "") {
```

Comment:
I think it might be a bit cleaner to add 1 line - `view.certificateInfo.isVisible = cert.isNotEmpty()` to the `bindCertificateName` fun?

Reply:
Good idea!  Done!